src/test/java/com/google/api/codegen/testdata/python_doc_types_no_path_templates.baseline

```diff
@@ -0,0 +1,39 @@
+============== file: google/cloud/proto/example_v1/types.py ==============
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import absoulute_import
+import sys
+
+from google.gax.utils.messages import get_messages
+
+from google.api import http_pb2
+from google.cloud.proto.example.v1 import no_path_templates_pb2
+from google.protobuf import descriptor_pb2
+from google.protobuf import empty_pb2
+
+
+names = []
+for module in (http_pb2
+               no_path_templates_pb2
+               descriptor_pb2
+               empty_pb2):
+    for name, message in get_messages(module).items():
+        message.__module__ = 'google.cloud.proto.example_v1'
```

Comment:
I would expect this to be `message.__module__ = 'google.cloud.example_v1'`. But I can not tell if it is wrong since it is hard to determine that from the bogus examples.